app/views/users/list.html.erb

```diff
@@ -1,11 +1,14 @@
+<div class='row'>
```

Comment:
Hi, I believe this div has to go inside the `col-md-3` div -- that way it appears inside the column, not above it. Thanks!